src/task/tests/config.ini

```diff
@@ -0,0 +1,38 @@
+; Licensed to the Apache Software Foundation (ASF) under one
+; or more contributor license agreements.  See the NOTICE file
+; distributed with this work for additional information
+; regarding copyright ownership.  The ASF licenses this file
+; to you under the Apache License, Version 2.0 (the
+; "License"); you may not use this file except in compliance
+; with the License.  You may obtain a copy of the License at
+;
+;   http://www.apache.org/licenses/LICENSE-2.0
+;
+; Unless required by applicable law or agreed to in writing,
+; software distributed under the License is distributed on an
+; "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+; KIND, either express or implied.  See the License for the
+; specific language governing permissions and limitations
+; under the License.
+
+[apps..default]
+type = test
+run = true
+count = 1
+pools = THREAD_POOL_DEFAULT, THREAD_POOL_TEST_SERVER
+
+[apps.client]
+arguments = localhost 20101
+ports =
+delay_seconds = 1
+
+[apps.server_group]
+arguments =
+ports = 20201
+count = 3
+
+[core]
+tool = simulator
+pause_on_start = false
+logging_start_level = LOG_LEVEL_DEBUG
+logging_factory_name = dsn::tools::screen_logger
```

Comment:
Should `THREAD_POOL_FOR_TEST_1` and `THREAD_POOL_FOR_TEST_2` be configured ?

Reply:
I missed this. Now both `simulator` and `nativerun` for `[core]tool` have been added to run tests.